defold-rive/pluginsrc/plugin.cpp

```diff
@@ -45,7 +45,8 @@ __declspec(dllexport) int dummyFunc()
 DM_STATIC_ASSERT(sizeof(dmGraphics::CompareFunc) == sizeof(int), Invalid_struct_size);
 DM_STATIC_ASSERT(sizeof(dmGraphics::StencilOp) == sizeof(int), Invalid_struct_size);
 // Making sure the size is the same for both C++ and Java
-DM_STATIC_ASSERT(sizeof(dmRive::RenderObject) == 288, Invalid_struct_size);
+DM_STATIC_ASSERT(sizeof(dmRive::ShaderConstant) <= 528, Invalid_struct_size);
+DM_STATIC_ASSERT(sizeof(dmRive::RenderObject) <= 8608, Invalid_struct_size);
```

Comment:
The plugin "renderobject" struct grew form 288 to 8608 bytes!
We can optimize it later, if we rewrite the shader constant wrappers a bit.